go.mod

```diff
@@ -1 +1,3 @@
 module github.com/platformsh/config-reader-go
+
+go 1.12
```

Comment:
It should technically work fine with Go 1.11, no?

Reply:
Actually don't think I added that. Generated?

Reply:
Just checked, this was generated when I ran `go test` locally.

Reply:
Sigh.  Go.